test/integration/epacems_test.py

```diff
@@ -67,15 +67,25 @@ def test_epacems_subset_input_validation(epacems_year_and_state, epacems_parquet
             epacems(epacems_path=path, **combo)
 
 
-def test_epacems_parallel(pudl_settings_fixture, pudl_ds_kwargs):
+def test_epacems_parallel(pudl_settings_fixture, pudl_ds_kwargs, tmpdir_factory):
     """Test that we can run the EPA CEMS ETL in parallel."""
     epacems_settings = EpaCemsSettings(
-        years=[2019, 2020], states=["ID", "WY"], partition=True
+        years=[2019, 2020], states=["ID", "ME"], partition=True
     )
-    etl_epacems(epacems_settings, pudl_settings_fixture, pudl_ds_kwargs)
+    # We need a temporary output directory to avoid dropping the ID/ME 2019/2020
+    # parallel outputs in the real output directory and interfering with the normal
+    # monolithic outputs.
+    parquet_tmp = tmpdir_factory.mktemp("parquet_tmp")
+    epacems_tmp = Path(parquet_tmp, "epacems")
+    epacems_tmp.mkdir()
```

Comment:
shouldn't we be doing this temp dir stuff in the generation of the `pudl_settings_fixture`?

Reply:
No, because we don't want to retain these parallelized outputs -- it's just for testing that the parallel version of the EPA CEMS ETL works.  If we don't use a different disposable place, they get mixed in with the real outputs. And if you run with `--live-dbs` they get mixed in with your actual EPA CEMS outputs. This causes problems with directory-level use of `pd.read_parquet()` since you then have duplicate versions of the same records sitting in multiple different parquet files in the same directory.

Reply:
hm. okay. and this is because this is a *secondary* cems test.